cbf/v1/cbf-v1.0.0.yaml

```diff
@@ -0,0 +1,591 @@
+        unitsNonDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units non DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units non DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+        unitsDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+        discharge:
+          type: array
+          description: Lists all the `LADEN` containers being discharged.
+          items:
+            $ref: '#/components/schemas/DischargeLadenContainer'
+    DischargeLadenContainer:
+      title: Discharge Laden Container
+      type: object
+      x-stoplight:
+        id: eba602e7e427f
+      properties:
+        portVisitReference:
+          type: string
+          description: The unique reference that can be used to link different transportCallReferences to the same port visit. The reference is provided by the port to uniquely identify a port call.
+          example: NLRTM1234589
+          pattern: ^\S+(\s+\S+)*$
+          maxLength: 50
+        transportCallReference:
+          type: string
+          description: "A carrier definied reference to a TransportCall.\r\n\r\nIn the case the Means of Transport is a Vessel and the facility is a Port/Terminal - this reference should be considered a Terminal Call Reference\r\n"
+          example: N987G4567
+          maxLength: 100
+          pattern: ^\S+(\s+\S+)*$
+        location:
+          $ref: ../../models/FacilitySMDGLocation.yaml
+        equipmentSizeCode:
+          type: string
+          enum:
+            - '20'
+            - '40'
+            - '45'
+            - 20HC
+            - 40HC
+            - 45HC
+          example: '20'
+          description: A CBF specification for a particular equipment size and type.
+        averageGrossWeight:
+          type: number
+          format: float
+        unitsNonDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units non DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units non DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
+        unitsDG:
+          type: object
+          properties:
+            total:
+              type: integer
+              description: Total units DG
+            totalPlugs:
+              type: integer
+              description: Subtotal units DG with plugs (reefer)
+          required:
+            - total
+            - totalPlugs
```

Comment:
I think it would be a good idea to split this out into a $ref since it is also specified further up in the structure

Reply:
Going to stick to the general rule of thumb, if it's not duplicated more than or equal to 3 times, leave it as is.